.env.test

```diff
@@ -0,0 +1,14 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+TRELLO_BOARD_ID='658ec5346800f22eb024ccec'
```

Comment:
This isn't secretive, but I'm still a fan of using really obviously fake variables - if nothing else it will help us spot if real ones sneak in:
```
TRELLO_BOARD_ID=fake-board-id
```